micrometer-tracing/src/main/java/io/micrometer/tracing/Span.java

```diff
@@ -71,6 +71,21 @@ public Span tag(String key, String value) {
             return this;
         }
 
+        @Override
+        public Span tag(String key, long value) {
+            return this;
```

Comment:
I think that the default could be just converting this to strings. `return tag(key, String.valueOf(value))`. WDYT?

Reply:
This is good suggestion, I applied it for all `Brave*` classes because brave really don't support this. For noops I prefer to  use shortest possible form